blobfuse/src/BlockBlobBfsClient.cpp

```diff
@@ -100,15 +100,17 @@ std::shared_ptr<blob_client_wrapper> BlockBlobBfsClient::authenticate_blob_accou
         {
             blobClient = std::make_shared<blob_client>(
             account,
-            configurations.concurrency);
+            configurations.concurrency,
+            config_options.debug_libcurl);
         }
         else
         {
             blobClient = std::make_shared<blob_client>(
             account,
             configurations.concurrency,
             configurations.caCertFile,
-            configurations.httpsProxy);
+            configurations.httpsProxy,
```

Comment:
Instead of modifying so many layers, we could just declare a global variable that's accessible to cpplite lib and use that instead. Make the changeset very small. There is already something similar done to dump REST requests in cpplite.

Reply:
uuuh, is there an example of this somewhere?

Reply:
Try following "gEnableLogsHttp" variable. We declare that in blobfuse binary while it's used inside cpplite using 'extern' which means while building we are just saying this will be declared later by some application using the lib. Based on our config we can just set the value and use it inside the lib.

Reply:
Current change set is also clean, it's just that touching so many files at this stage where things are pretty stable and we are not doing many changes in code, it's not safe.